### ISSUE TITLE
Pipeline crash during partition

### ISSUE THREAD

**Issue by amandaj1430** [hillerlab/make_lastz_chains#14]
Hello,

I'm trying to create chain alignments for whole primate genomes (novel sequences) against the vervet reference genome. I removed the spaces and tabs from the fasta file while allowed the process to being but now the pipeline is crashing at doPartition.bash 

`dwnloads/make_lastz_chains/make_chains.py --pd chain_files --force_def --DEF chain_files/DEF sabaeus neglectus chlSab1/vervet_nospace.fa GuenH/assembly.megabubbles.ns.fasta`

Here's the log file: [do partition log file.pdf](https://github.com/hillerlab/make_lastz_chains/files/11064429/do.partition.log.file.pdf)

Please let me know what suggestions you have to fix this error.

**Comment by amandaj1430** [comment]
Apologies, I attached the wrong log file. Please see the correct one here: 
[Make_lastz_chains logfile 2.pdf](https://github.com/hillerlab/make_lastz_chains/files/11064617/Make_lastz_chains.logfile.2.pdf)

**Comment by danielle-khost** [comment]
Hi there, I was having the same issue! I _believe_ the problem is the ":" characters in the headers, as the pipeline sees what is after the colon as coordinates and gets confused. I was able to get it running by replacing the ":" characters using `sed`:
`sed -i 's/:/#/g' Gallus_gallus.bGalGal1.mat.broiler.GRCg7b.dna_sm.toplevel.fa`

Hope that helps! 

(I also noticed the pipeline is particular about very long headers as well...I had to trim down the headers in a genome I downloaded off Ensembl in order for it to not crash)
